docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -200,4 +201,46 @@ If there's a match, we remove the false positive from the list.
 
 - CentOS 6, 7 and 8.
 
+## What Fleet checks for vulnerabilites
```

Comment:
```suggestion
## Coverage
```

What do you think about calling this section something like "Coverage" ? This way, these docs clearly answer the following question: As a Fleet user, what I can rely on Fleet for in terms of coverage ?

The goal we're trying to achieve with these doc updates is the following (taken from #4928):
> As a Fleet user, I want to understand what installed software Fleet collects so that I know what I can rely on Fleet for in terms of coverage when comparing Fleet to other vulnerability management tools.